### ISSUE TITLE
Missing "quotation mark" - mssql_cli stuck

### ISSUE THREAD

**Issue by Komport** [dbcli/mssql-cli#136]
I am testing simple stuff with mssql_cli on Centos 7.0 and Python 2.7. When i am doing simple insert command as below in one point I forget to put closing quotation mark after some value. mssql_cli stuck at this point and only Ctrl+C can help.
```
DBSIZE>INSERT INTO test_table (some_value) values('Testing);
^CCancelling query...
Query cancelled.
DBSIZE>INSERT INTO test_table (some_value) values("Testing);
```

**Comment by Komport** [comment]
After İ have waited a little longer and hit Ctrl+C I got below TraceBack

```
DBSIZE>INSERT INTO test_table (some_value) values("Testing);
";
^CCancelling query...
Traceback (most recent call last):
  File "/usr/lib64/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/lib/python2.7/site-packages/mssqlcli/main.py", line 867, in <module>
    cli()
  File "/usr/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/mssqlcli/main.py", line 736, in cli
    mssqlcli.run_cli()
  File "/usr/lib/python2.7/site-packages/mssqlcli/main.py", line 413, in run_cli
    query = self.execute_command(document.text, query)
  File "/usr/lib/python2.7/site-packages/mssqlcli/main.py", line 342, in execute_command
    self.mssqlcliclient_query_execution)
  File "/usr/lib/python2.7/site-packages/mssqlcli/mssqlcliclient.py", line 304, in reset_connection_and_clients
    raise e
OSError: [Errno 3] No such process

```

**Comment by MrMeemus** [comment]
Thanks for reporting, it could be a mishandled error from sqltoolsservice leaving the process hanging. Will take a look.

**Comment by MrMeemus** [comment]
Fixed in #140
